packages/atomic/playwright-utils/base-page-object.ts

```diff
@@ -0,0 +1,43 @@
+import type {Page} from '@playwright/test';
+import {buildArgsParam} from '@storybook/router';
+import {JSX} from '../dist/types/components';
+
+export class BasePageObject<
+  TagName extends keyof JSX.IntrinsicElements,
+  Component = JSX.IntrinsicElements[TagName],
+> {
+  constructor(
+    public page: Page,
+    public tag: TagName
+  ) {}
+
+  get hydrated() {
+    return this.page.locator(`${this.tag}[class*="hydrated"]`);
+  }
+
+  get urlRoot() {
+    return 'http://localhost:4400/iframe.html';
+  }
+
+  async load(args?: Component, story: string = 'default') {
+    if (args) {
+      await this.page.goto(
+        `${this.urlRoot}?id=${this.tag}--${story}&args=${buildArgsParam(undefined, this.camelToKebab(args))}`
+      );
+    } else {
+      await this.page.goto(`${this.urlRoot}?id=${this.tag}--${story}`);
+    }
+    await this.hydrated.waitFor();
```

Comment:
That's my opinion, but I see the "wait on hydration" as an escape hatch and an escape hatch only: I think it should be used when we have no other means of awaiting, not as a "standard."

Hydration is an internal concept and does not reflect "what the user sees and does" like the other locators do.
I would remove:
```suggestion
```

And add it as needed, not everywhere.